src/main/java/com/singlestore/fivetran/destination/writers/Writer.java

```diff
@@ -3,10 +3,7 @@
 import com.github.luben.zstd.ZstdInputStream;
 import com.google.protobuf.ByteString;
 import com.opencsv.CSVReader;
-import fivetran_sdk.Compression;
-import fivetran_sdk.CsvFileParams;
-import fivetran_sdk.Encryption;
-import fivetran_sdk.Table;
+import fivetran_sdk.*;
```

Comment:
why did you change explicit import to *?

Reply:
It was done automatically by the IDE. Changed back